libkbfs/kbfs_ops.go

```diff
@@ -1115,11 +1148,13 @@ func (fs *KBFSOpsStandard) NewNotificationChannel(
 		fs.editActivity.Add(1)
 		go func() {
 			defer fs.editActivity.Done()
-			fs.log.CDebugf(ctx, "Initializing TLF %s for the edit history",
-				handle.GetCanonicalPath())
+			fs.log.CDebugf(ctx, "%p: Initializing TLF %s for the edit history",
```

Comment:
Why output the pointer address?

Reply:
Just extra debugging I left in.  Will remove, thanks!